src/main/java/lib/controllers/Ex3DProController.java

```diff
@@ -41,8 +41,8 @@ public enum Ex3DProAxis {
     }
   }
 
-  public JoystickButton get(Ex3DProButton button) {
-    return new JoystickButton(m_controller, button.id);
+  public Trigger get(Ex3DProButton button) {
+    return new Trigger(() -> m_controller.getRawButton(button.id));
```

Comment:
I might be wrong about this but pretty sure the Ex3Dpro has a hat stick. You should add it in this PR as it is already pretty low priority so there is no rush.